### PR TITLE
Axel

### PR DESCRIPTION
**Pull request by hauduc**
Re-did pull request for Tutorial 1 with typo fix!

### CODE REVIEW

File: Rmds/windows.Rmd [hunk 1]
```diff
@@ -18,13 +18,23 @@ Please update to the latest version of Windows 10: version `r params$windows_ver
...
 
--   Go to [Microsoft Store Windows Terminal website](https://aka.ms/terminal) > click the "Get" button to install the app.
+-   The simplest way to access OpenSSH on Windows is to open the "Command Prompt" application, discoverable through the search bar. This application will provide sufficient functionality for all server-based work you will do in MICB405.
+
+-   Once here, you can proceed to Tutorial 1.
+
+## Additional considerations for Windows {-}
+
+-   Be aware that on Windows Command Prompt, you will be accessing a Linux server through a multi-platform software package, OpenSSH, that exists in both Windows and bash/Linux/Unix. Functionality will be identical once you are connected, but before you are connected, your terminal is entirely different from bash/Linux/Unix and [uses a different language](http://www.cs.columbia.edu/~sedwards/classes/2015/1102-fall/Command%20Prompt%20Cheatsheet.pdf). Tread with caution if executing commands other than `ssh` on Command Prompt!!
```
Comment (stephan-koenig) [inline Rmds/windows.Rmd:33]:
```suggestion
Be aware that when you open a new Windows Terminal window that you will initially use an entirely different shell/language (called [PowerShell](https://docs.microsoft.com/en-us/powershell/)) on your computer. Only after you are connected to the server (using OpenSSH, see above) will you be using Bash. Tread with caution if executing commands other than `ssh` or `scp` from the tutorials while still in PowerShell (i.e. while not connected to the server).
```

File: Rmds/windows.Rmd [hunk 2]
```diff
@@ -18,13 +18,23 @@ Please update to the latest version of Windows 10: version `r params$windows_ver
...
+
+-   If you would like to make your terminal more customizeable and easy to use, you can download [Windows Terminal](https://aka.ms/terminal) (click the "Get" button to install the app) and access Command Prompt through there.
+
+-   Finally, you can create a true Linux environment at the command line by setting up [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10). You can then access this virtualized Linux shell on Windows Terminal, and `ssh` onto the server as you would on a Linux machine's terminal. This is entirely optional, but will allow you practice and run Linux commands on your machine without connecting to the server, just like on macOS and Linux.
```
Comment (stephan-koenig) [inline Rmds/windows.Rmd:37]:
```suggestion
## Optional: Windows Subsystem for Linux

You can create a true Linux environment at the command line by setting up [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10). You can then access Bash of this virtualized Linux to practice and run Linux commands on your machine without connecting to the server.
```

**Review by stephan-koenig — COMMENTED** [review]
Hi Axel,

Please do not close and re-open a Pull Request. You can commit to your branch, and the PR will show the latest commits. I had just been working on your first pull request, and after you closed it, my comments were gone.

Please pay attention to what files you commit to the repo. There should be only a single `.Rproj` file, and you should not save your history to the repo (`-MSL-HIRST-0014.Rhistory`). You also committed a failed rendering of the main document `_main.Rmd`.

I had a very negative experience with PowerShell last year. I have tested Windows Terminal, and it was very similar to the Mac Terminal experience. I do not know if the command prompt acts similarly. For that reason, I think Windows Terminal should be our default. It also makes it easier to say "Open your terminal" since all apps have terminal in their name.

I will add now my own edits before I merge.

Thanks,

Stephan

**Review by stephan-koenig — APPROVED** [review]

**Comment by hauduc** [comment]
Hi Stephan,

PowerShell will behave identically whether accessed through the standalone Windows app or through Windows Terminal though no? I don't want to make things too complicated because I had students with issues with Windows Terminal last year. If they can open Command Prompt or PowerShell on the standalone apps, they can use ssh and I just wanted to emphasize to then that that is sufficient.

Will be careful with which files to commit in the future.

What was the issue with the rendering of _main? I just saw that the bookdown is deployed though, looks great!

Best,
Axel
________________________________

**Pull request merged**